mods/ra/rules/defaults.yaml

```diff
@@ -626,17 +626,16 @@
 		RequiresCondition: cruising
 	BodyOrientation:
 		UseClassicFacingFudge: True
-
+	
 ^BasicBuilding:
 	Inherits@1: ^ExistsInWorld
 	Inherits@2: ^IronCurtainable
 	Inherits@3: ^SpriteActor
 	Inherits@shape: ^1x1Shape
 	Inherits@bounty: ^GlobalBounty
+	Inherits@SELECTION_TIER: ^SelectableBuilding
```

Comment:
`SELECTION_TIER` doesn't seem to fit the other suffices. What about just using `@selection`?

Reply:
That name was suggested by me, because I am proposing in #15839 a `Inherits@SELECTION_MODE: ^FlatSelectionMode`. Though these templates could be combined into a single template.

Reply:
I believe @abcdefg30 meant the `SELECTION_TIER` name, vs `1`, `shape`, `bounty`, etc.

Reply:
That's what I meant as well :-) We can combine `SELECTION_MODE` and `SELECTION_TIER` into the same declaration (if the other PR gets merged). So we'd then have for example:
```
^SelectableSupportUnit:
	Selectable:
		Priority: 8
		PriorityModifiers: Ctrl, Alt
```
```
	Inherit@SELECTION: ^SelectableSupportUnit
```

Reply:
I'm just asking about the upper case and the underscore - not common standard afaik.

Reply:
I'll make these changes over the weekend. Thanks for your feedback!

Reply:
I thought that uppercase-underscore is the convention in OpenRA for such cases 🤔 it certainly is in chrome yamls.

Reply:
Yes, those are different. ^^)